rof-scheduler/src/CreateClient/index.js

```diff
@@ -0,0 +1,351 @@
+
+  const togglePassword = () => {
+    setShowPassword(!showPassword);
+  };
+
+  const handleSubmit = (submitEvent) => {
+    submitEvent.preventDefault();
+
+    const countryId = 1;
+    const firstName = submitEvent.target.firstName.value;
+    const lastName = submitEvent.target.lastName.value;
+    const emailAddress = submitEvent.target.emailAdd.value;
+    const username = submitEvent.target.username.value;
+    const password = submitEvent.target.password.value;
+    const addressLine1 = submitEvent.target.address1.value;
+    const addressLine2 = submitEvent.target.address2.value;
+    const city = submitEvent.target.city.value;
+    const state = submitEvent.target.state.value;
+    const zipCode = submitEvent.target.zip.value;
+    const primaryPhoneNum = submitEvent.target.primPhone.value;
+    const secondaryPhoneNum = submitEvent.target.secPhone.value;
+
+    const validationRes = ensureCreateClientInformationProvided(
+      firstName,
+      lastName,
+      emailAddress,
+      username,
+      password,
+      primaryPhoneNum,
+      addressLine1,
+      city,
+      state,
+      zipCode
+    );
+
+    if (validationRes.size > 0) {
+      setValidationMap(validationRes);
+    } else {
+      setValidationMap(new Map());
+      setLoading(true);
+
+      (async function () {
+        try {
+          await createClient(
+            countryId,
+            firstName,
+            lastName,
+            emailAddress,
+            username,
+            password,
+            primaryPhoneNum,
+            secondaryPhoneNum,
+            addressLine1,
+            addressLine2,
+            city,
+            state,
+            zipCode
+          );
+
+          setClient({
+            ...client,
+            firstName: submitEvent.target.firstName.value,
+            lastName: submitEvent.target.lastName.value,
+            emailAddress: submitEvent.target.emailAdd.value,
+            username: submitEvent.target.username.value,
+            password: submitEvent.target.password.value,
+            primaryPhoneNum: submitEvent.target.primPhone.value,
+            secondaryPhoneNum: submitEvent.target.secPhone.value,
+            addressLine1: submitEvent.target.address1.value,
+            addressLine2: submitEvent.target.address2.value,
+            city: submitEvent.target.city.value,
+            state: submitEvent.target.state.value,
+            zipCode: submitEvent.target.zip.value,
+          });
+
+          setErrMsg("");
+        } catch (e) {
+          setErrMsg(e.message);
+        } finally {
+          setLoading(false);
+        }
+      })();
+    }
+  };
+
+  return (
+    <>
+      <Container
+        style={{
+          display: "flex",
+          alignItems: "center",
+          justifyContent: "center",
+          paddingTop: "2rem",
+        }}
+      >
+        <Form noValidate onSubmit={handleSubmit}>
+          {errMsg !== "" && <Alert variant="danger">{errMsg}</Alert>}

```

Comment:
Once successful, display a positive feedback (similar what i did in account settings last commit).

In the successful message, tell them - "You have successfully signed up. Click here to login".

Click here will redirect them to login page.